spartan/aztec-network/values/48-validators.yaml

```diff
@@ -9,7 +9,6 @@ telemetry:
   otelCollectorEndpoint: http://metrics-opentelemetry-collector.metrics:4318
 
 validator:
-  debug: "aztec:*,-aztec:avm_simulator:*,-aztec:libp2p_service"
```

Comment:
Does this level of logging causing stability issues? (for my own understanding)

Reply:
We've seen issues in e2e tests where excessive logging messes up tests. I haven't seen it in network tests as far as I am aware. 

However, this was actually reducing the level of logs, since the default level in values.yaml has more negative filters.